package.json

```diff
@@ -38,8 +38,11 @@
     "eslint-config-scratch": "^3.0.0",
     "eslint-plugin-react": "^7.0.1",
     "gh-pages": "github:rschamp/gh-pages#publish-branch-to-subfolder",
+    "grapheme-breaker": "0.3.2",
     "html-webpack-plugin": "2.28.0",
+    "ify-loader": "1.0.4",
```

Comment:
The `grapheme-breaker` and `linebreak` modules use `fs` in such a way that Webpack can't handle them directly. I found the suggestion to use `ify-loader` here: devongovett/linebreak#1